### ISSUE TITLE
[Bug] [seatunnel-examples] LocalSparkExample don't set deployMode

### ISSUE THREAD

**Issue by kone-net** [apache/seatunnel#1614]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-seatunnel/issues?q=is%3Aissue+label%3A%22bug%22) and found no similar issues.

### What happened

When I run LocalSparkExample in our local, and got NullPointerException issue.

The reason is that LocalSparkExample is not set deploy mode.

I checked the reversion code and found the default deployMode value has been removed in SparkCommandArgs.

### SeaTunnel Version

dev branch

### SeaTunnel Config

```conf
the default config for spark.batch.conf in examples
```

### Running Command

```shell
Debug in the idea and no other parameters
```

### Error Exception

```log
Exception in thread "main" java.lang.NullPointerException
	at org.apache.seatunnel.Seatunnel.run(Seatunnel.java:41)
	at org.apache.seatunnel.example.spark.LocalSparkExample.main(LocalSparkExample.java:35)
```

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by BenJFan** [comment]
Seem like you already fix it, please create an PR.

**Comment by CalvinKirs** [comment]
closed by #1615
